apps/cloud_inference/server_app/CMakeLists.txt

```diff
@@ -30,15 +30,10 @@ set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../../amd_openvx/cmake)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
 
-find_package(OpenCL QUIET)
```

Comment:
Need to do OpenCV version related changes for supporting both versions(3&4) of OpenCV